Cargo.toml

```diff
@@ -184,3 +184,34 @@ debug = true
 rocksdb = { git = 'https://github.com/aptos-labs/rust-rocksdb' }
 serde-reflection = { git = "https://github.com/aptos-labs/serde-reflection", rev = "839aed62a20ddccf043c08961cfe74875741ccba" }
 diesel = { git = "https://github.com/aptos-labs/diesel", branch = "1.4.x" }
+
+# Replacing original dependences of "move-language/move"
+[patch."https://github.com/move-language/move"]
+move-abigen = { git = "https://github.com/bobchainer/move", branch = "call-traces" }
```

Comment:
better to link to commit

Reply:
I think it's better for libraries where we don't the `Cargo.lock` file. In this repo, we do have a commit locked in the lock file and the only way to update it is running `cargo update`

Reply:
Resolved during a standup. It's fine to leave a branch name for now.